scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -572,30 +590,37 @@ function Reset-SystemPath {
 
 function Get-VcRuntime {
     if (Test-IotCore) {
-        Write-HostGreen 'Skipped vcruntime download on IoT Core.'
+        Write-HostGreen 'Skipping VC Runtime installation on IoT Core.'
```

Comment:
I believe there is an issue (#565) that is asking if we can make this optional?

Reply:
https://github.com/Azure/iotedge/pull/592/commits/54da4c803097001edb0a7ec1d022c2a5fac9f0b0

I'll add the `Fixes ` to the commit when I merge it.